pom.xml

```diff
@@ -26,8 +26,7 @@
         <changelist>999999-SNAPSHOT</changelist>
         <gitHubRepo>jenkinsci/build-with-parameters-plugin</gitHubRepo>
         <revision>1.7</revision>
-        <changelist>-SNAPSHOT</changelist>
```

Comment:
Wasn't sure what to do with this duplicate property. If it's intentional I'd like to know more about it so I can handle this case in the tooling.